sdcm/cluster.py

```diff
@@ -3192,6 +3192,18 @@ def __init__(self, cluster_uuid=None, cluster_prefix='cluster', node_prefix='nod
             self.run_node_benchmarks()
         self.coredumps = {}
         super().__init__()
+        self.keyspace_info = {}
+
+    def get_keyspace_info(self, keyspace_name: str, db_node: BaseNode):
+        # Prevent circular import
+        from sdcm.utils.replication_strategy_utils import ReplicationStrategy  # pylint: disable=import-outside-toplevel
+
+        if self.keyspace_info.get(keyspace_name, None) is None:
+            replication_strategy = ReplicationStrategy.get(db_node, keyspace_name)
+            self.keyspace_info[keyspace_name] = {"replication_factors": replication_strategy.get_replication_factors(),
```

Comment:
I think we could simplify it further, by removing dict from value:
`self.keyspace_info[keyspace_name] = replication_strategy`
so later could be used:
```
replication_strategy = self.get_keyspace_info(keyspace_name, node)
return set(replication_strategy.replication_factors) == {1}
```

Reply:
changed